tests/SimplifyTest.elm

```diff
@@ -12,6 +12,7 @@ all =
         [ configurationTests
         , qualifyTests
         , lambdaReduceTests
+        , operatorRangeTests
```

Comment:
Maybe move this to the bottom of the list, since this is more of a test about the implementation than about a specific transformation.
I'd also be happy with moving the new tests inline next to the tests for `"Element added to the beginning of the list could be included in the list"`. That way when v8 comes around there won't be a good reason to have a dedicated describe.